xl-v1-metadata.go

```diff
@@ -52,6 +52,12 @@ type checkSumInfo struct {
 	Hash      string `json:"hash"`
 }
 
+// Default bit-rot checksum technique used in version '1'.
+const (
+	bitRotAlgoV1 = "blake2b"
```

Comment:
> V here stands for version. "sha512" will be a different algo from "blake2b" so it can't be a version increment.
> 
> Did you mean to call them bitRotAlgo1 and bitRotAlgo2?
> 
> if V version is for xl format version, even then xl version would still be same so calling it bitRotAlgoV2 will be inappropriate

Actually you are right it won't make sense.. Since we won't bump `xl.json` .